overlay/files/etc/cos/bootargs.cfg

```diff
@@ -0,0 +1,10 @@
+set kernel=/boot/vmlinuz
+if [ -n "$recoverylabel" ]; then
+    # Boot arguments when the image is used as recovery
+    set kernelcmd="console=tty1 root=live:CDLABEL=$recoverylabel rd.live.dir=/ rd.live.squashimg=$img panic=5"
+else
+    # Boot arguments when the image is used as active/passive
+    set kernelcmd="console=tty1 root=LABEL=$label net.ifnames=1 iso-scan/filename=$img panic=5 security=selinux rd.cos.oemlabel=COS_OEM selinux=1"
```

Comment:
were these se linux args already there?

Reply:
I copied the default elemental bootargs.cfg from documentation and added net.ifnames=1 
Default values copied from here
https://rancher.github.io/elemental-toolkit/docs/customizing/configure_grub/